src/scripts/shell/platform-choice/platform-choice.component.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
```

Comment:
Maybe we should hold off and switch to lodash all in one go. Otherwise we'll end up bundling both lodash AND underscore.